src/plone/api/content.py

```diff
@@ -473,3 +474,92 @@ def get_uuid(obj=None):
     :Example: :ref:`content_get_uuid_example`
     """
     return IUUID(obj)
+
+
+def find(context=None, depth=None, **kwargs):
+    """Find content in the portal.
+
+    Find works alike catalog(). Indexes are passing in as arguments with the
```

Comment:
I find this doc string overly descriptive. When I check the api doc strings, I just want to see a brief description and the params. Narrative documentation like this should go into the docs.